aopy/analysis.py

```diff
@@ -147,10 +159,11 @@ def run_curvefitting(means, make_plot=True, fig_title='Tuning Curve', n_subplot_
         n_cols (int) : No. of cols in subplot
 
     Returns:
-        tuple: Tuple containing:
-            | **params_day (Numpy array):** Curve fitting parameters
-            | **modulation depth (Numpy array):** Modulation depth of neuron
-            | **preferred direction (Numpy array):** preferred direction of neurons
+        params_day (Numpy array) : Curve fitting parameters
+        modulation depth (Numpy array) : Modulation depth of neuron
+        preferred direction (Numpy array) : preferred direction of neurons
+
+    See Also: :func:`aopy.analysis.tuning_fit_func` , :func:`aopy.analysis.get_preferred_direction`, :func:`aopy.analysis.get_modulation_depth`
```

Comment:
An example here would be appropriate. Maybe more useful than an example for any of the helper functions.